clrdefinitions.cmake

```diff
@@ -236,3 +236,5 @@ if(WIN32)
     add_definitions(-DFEATURE_DATABREAKPOINT)
   endif(CLR_CMAKE_TARGET_ARCH_AMD64 OR CLR_CMAKE_TARGET_ARCH_I386)
 endif(WIN32)
+
+add_definitions(-DFEATURE_BASICFREEZE)
```

Comment:
Could you please put it earlier in the file where other unconditional defines are?

Reply:
Yup, put it where it says Features should be put in sorted order.